paper.tex

```diff
@@ -539,7 +538,7 @@ \subsection{Pipeline Design} \label{subsec:pipeline-design}
 \label{alg:flatten-cubic}
 \end{algorithm}
 
-All input segments first get converted to a cubic Bézier by degree elevation. Our flattening routine (see Algorithm~\ref{alg:flatten-cubic}) starts by computing the error estimate for geometric Hermite interpolation from the cubic Bézier to an Euler spiral segment. If the error exceeds the desired tolerance threshold, the cubic segment is iteratively subdivided in half and a new error estimate is computed. Once the error satisfies the tolerance, the Euler spiral segment is directly flattened to a polyline by invoking a subroutine that is parameterized by the stroke half-width (see \texttt{es\_seg\_flatten\_offset} in Algoritm~\ref{alg:flatten-cubic}). For a regular fill, this parameter is set to $0$. For strokes, the sign of the parameter determines the position of the parallel curve relative to the source segment as well as the winding direction of the output lines.
+All input segments first get converted to a cubic Bézier by degree elevation. Our flattening routine (see Algorithm~\ref{alg:flatten-cubic}) starts by computing the error estimate for geometric Hermite interpolation from the cubic Bézier to an Euler spiral segment. If the error exceeds the desired tolerance threshold, the cubic segment is iteratively subdivided in half and a new error estimate is computed. Once the error satisfies the tolerance, the Euler spiral segment is directly flattened to a polyline (or, alternately, converted to a sequence of arc segments) by invoking a subroutine that is parameterized by the stroke half-width (see \texttt{es\_seg\_flatten\_offset} in Algoritm~\ref{alg:flatten-cubic}). For a regular fill, this parameter is set to $0$. For strokes, the sign of the parameter determines the position of the parallel curve relative to the source segment as well as the winding direction of the output lines.
```

Comment:
I actually intentionally removed the "(or, alternately, converted to a sequence of arc segments)" part because we later say "We also simplemented a variant that outputs circular arc segments instead of lines" a few paragraphs down.